lms/product/plugin/misc.py

```diff
@@ -1,11 +1,14 @@
-from typing import NotRequired, TypedDict
+from typing import TYPE_CHECKING, NotRequired, TypedDict
 
 from pyramid.request import Request
 
 from lms.js_config_types import AutoGradingConfig
-from lms.models import Assignment, LTIParams, LTIRegistration
+from lms.models import Assignment, LTIParams
 from lms.services.html_service import strip_html_tags
 
+if TYPE_CHECKING:
```

Comment:
Trying to fix circular dependency with `Family`.

Didn't success but leaving this in.